src/NumericalAlgorithms/Interpolation/LinearLeastSquares.hpp

```diff
@@ -51,6 +52,19 @@ class LinearLeastSquares {
   std::array<double, Order + 1> fit_coefficients(const T& x_values,
```

Comment:
I can't insert a comment where I want, but what is `num_observations`?  Please document it